ui/app/templates/workspace/projects/project/app/deployments.hbs

```diff
@@ -2,9 +2,13 @@
   {{t "page.deployments.title"}}
 </h3>
 
-<ul data-test-deployment-list=true class="list">
-  {{#each this.deployments as |deployment|}}
-    <AppItem::Deployment @deployment={{deployment}} />
+<ul data-test-deployment-list class="list">
+  {{#each this.deploymentsByGeneration key="generationID" as |group|}}
+    {{#each group.deployments key="id" as |deployment|}}
+      {{#if (or this.isShowingDestroyed (not-eq deployment.state 4))}}
+        <AppItem::Deployment @deployment={{deployment}} @latest={{group.deployments.[0]}} />
+      {{/if}}
+    {{/each}}
   {{else}}
     <EmptyState>
       <p>There are no deployments to display for this app yet</p>
```

Comment:
Should this text be in the `translations/` file?

Reply:
I started moving some of the text in this `deployments.hbs` file to the translations files, but @jgwhite said the checklist item about it on this PR was meant for the single deployment item template, not this overarching deployments template.

Reply:
I added the other text to the translation file. Jamie and I discussed this specific block of text yesterday and agreed the paragraph breaks make it a bit more of a complex situation. It's a little out of scope for this PR, so we can add this text to the localization texts some other time.